# <div aling=center>mRNA多聚腺苷酸化调控AUXIN信号及生长发育的机制</div>

**Title: Modulation of Auxin Signaling and Development by Polyadenylation Machinary**

## ABSTRACT

&ensp;Polyadenylation influences gene expression by affecting mRNA stability, transport, and translatability. Alternative polyadenylation (APA) of mRNAs has emerged as an important mechanism for post-transcriptional gene regulation in higher eukaryotes. 

&ensp;The AtCstF77 protein was localized in the nucleus, which is consistent with its function in pre-mRNA processing. And this study observed that PAPs in transcripts from apporximately 2400 genes were shifted in the *cstf77-2* mutant.

* **APA can impact gene expression through multiple mechanisms**

  First, alternatively polyadenylated mRNAs can code for different protein isoforms with distinct physiological properties. (Like alternative splicing, APA contributes to the expansion of protein diversity in higher eukaryotes.)

  Secondly, APA generates mRNAs with distinct 3' UTRs. Many sequence elements found in extended 3' UTRs, such as AU rich and GU rich elements and microRNA target sites, have a largely negative influence on mRNA stability and/or translation effeciency. As results, mRNAs with shorter 3' UTRs tend to express higher levels of proteins.

  Thirdly, as mRNAs deavage/polyadenylation is intimately linked to other steps of gene expression, the choice of alternative poly(A) sites can influence other gene expression steps.

  Finally, APA of anti-sense RNAs can differentially silence the expression of a sense gene at least in part through modulating histone modifications in plants.

  In *Arabidopsis thaliana*, more than 70% of genes possess two or more PASs (Polydenylation sites).

* <p align = "middle">Chilling stress inhibits rRNA biogenesis mainly at pre-rRNAs processing levels</p>

* **The polyadenylation machinery cleaves the pre-mRNA at the polyadenylation site (PAS) and adds the poly(A) tail**

## RESULTS

* **Isolation of a sirtionl / Auxin-resistant mutant by EMS and molecular cloning of *AtCstF77***

  <div align=center>
<img src="https://upload-images.jianshu.io/upload_images/19253200-cff1f6a7d74c464e.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/463/format/webp">
  </div>

  <p align = "middle">Ethly-methanesulfonate-mutagenized Arabidopsis Ler population</p>

* **The *cstf77* mutants had reduced Auxin responses**

  <div align=center>
  <img src="https://upload-images.jianshu.io/upload_images/19253200-581eb53a9bd013d5.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/521/format/webp">
  </div>

  <sup>Ps: The **axr1-3** mutant is a strong auxin-resistant mutant<sup>

  Also use the **DR5-GFP** auxin response reporter to prove.

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-5fa9a86e43fed257.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/438/format/webp">
  </div>

  
* **AtCstF77 is ubiquitously expressed and localized in the nucleus proved by observe the location of CstF77-GFP and CstF77-GUS fusion protein, and the CstF77-GUS fusion protein was highly expressed in embryos**

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-09e965c5cc4e56ed.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/536/format/webp">
  </div>

* **Both CstF77 and AUXIN can affects the choice on Poly(A) site**

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-df5fa7d77b3d0873.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/580/format/webp">
  </div>

  <sub>其实这一部分我没有怎么看懂，因为下文也没有再给出相关性很强的实验结果，个人认为若要证明这一部分的结果，即要从这些不同的类型中挑选出几个关键的点，进行论证</sub>

  后续研究者做了一个RNAseq，结果发现在这些差异基因中，GO富集分析能够富集到激素响应相关的基因通路上。（RNAseq差异基因展示的图其实有很多种，远不止我们传统学习的火山图，我其实一直不认为火山图很好看，但是人家拿火山图也能发big paper，嘿嘿。这个GO富集分析的图还挺有意思的，可以学习一下）

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-bc5ab47a4dc93609.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/982/format/webp">
      <br>
      <br>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-3d68f3963e5782b1.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/1200/format/webp">
  </div>

  这一步RNAseq其实还是在对上面PAS-seq结果的一个论证，经过统计分析可以看出AtCstF77 和 AUXIN 都能够影响PAS。

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-00ea9be8fa29c2b6.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/305/format/webp">
  </div>

  最后研究者挑选了几个PAS shift的AUXIN相关的基因，对其表达量进行了查看（这里应该是用的PAS-seq的数据进行查看的，可以看到在 3' UTR 上会有某一段的升高，意味着可能在这个位点发生了PAS shift。

  <div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-c598f62cafab8acb.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/534/format/webp">
  </div>

  
最后，由于看这篇paper时不太懂PAS-seq，所以我又看了一下这篇文章引用的PAS-seq的paper（顺便翻了一下作者的主页，貌似挺牛的），展示一下PAS-seq的流程图，其实这边文章并不是很大，但是2011年的文章到现在已经引用的三百多次，所以还是不错的（告诉我们科学并不是要发多大的文章，而是你的发现真正对大家有用）。
  
<div align=center>
      <img src="https://upload-images.jianshu.io/upload_images/19253200-48c1cc4e070a7c28.jpg?imageMogr2/auto-orient/strip|imageView2/2/w/331/format/webp">
  </div>
  

  ***
  
  DOI: https://doi.org/10.1104/pp.18.00782
  
  DOI: https://www.rnajournal.cshlp.org/content/17/4/761